src/Compute.js

```diff
@@ -55,6 +58,21 @@ export default function Compute({ ocean, web3 }) {
   async function startCompute(algorithmId, algorithmMeta) {
     try {
       const accounts = await ocean.accounts.list()
+      const ComputeOutput = {
+        publishAlgorithmLog:
+          PublishLogState === '' || PublishLogState === false ? false : true,
```

Comment:
Just use `false` as a default value and you can get rid of all these ternaries

Reply:
Sometimes PublishLogState  can be ''   when unchecked (at least on my Firefox)

Reply:
Are you sure it's not because of the default value? I've checked on Firefox and `event.target.checked` is always a boolean (as it should per spec). Even if for some reason it would be an empty string we could just do `setPublishLogState(!!event.target.checked)` to cast it to a boolean

Reply:
yeah, default value was to blame